Docs: removing callouts and other updates (#1827)
vtex/faststore: apps/site/pages/docs/getting-started/5-next-steps.mdx

</header>

> Now that you have your FastStore project up and running, it's important to explore the additional resources to enhance your store experience:

Author: ```suggestion
Now that you have your FastStore project up and running, it is important to explore the additional resources to enhance your store experience:
```

> <Callout type="warning" emoji="⚠️">

Reviewer: I think we can remove this callout.

Author: Good point, Fanny ;)

> <Callout type="warning" emoji="⚠️">

Author: ```suggestion
```

> This documentation is currently under development.

Author: ```suggestion
```

> </Callout>

Author: ```suggestion
```

> Now that you have your FastStore project up and running, it's important to explore the additional resources to enhance your store experience:

Reviewer: Cool! 👍

## Theming

Modify the appearance and user interface of your store with [Theming](/docs/brandless). By customizing the theme, you can give your store a unique look and feel that aligns with your brand.

## VTEX Headless CMS (coming soon!)

Manage your store content with our preferred solution for content management. This will give editors the autonomy to create, edit, and publish web content via the VTEX Admin.

## Component customization

Explore the various ways you can extend and customize the components of your store by checking the [Component Customization track](/docs/component-customization/component-customization-overview). This could involve modifying existing components or creating new ones to meet your specific requirements. By customizing components, you can tailor your store's functionality to suit your business needs.

## UI components

Enhance your store's features with the [FastStore UI Components](/components). These components are designed to help you quickly implement and customize features in your store.

## Go-live

> Integrate your FastStore project with VTEX and get ready to launch your store with the [Go-live track](/docs/go-live/overview) . This involves connecting your store to VTEX's infrastructure, setting up payment gateways, configuring shipping options, and ensuring a seamless checkout experience for your customers.

Reviewer: ```suggestion
Integrate your FastStore project with VTEX and get ready to launch your store with the [Go-live track](/docs/go-live/overview). This involves connecting your store to VTEX's infrastructure, setting up payment gateways, configuring shipping options, and ensuring a seamless checkout experience for your customers.
```
